app/views/api/stripe_transactions/_stripe_transaction.json.jbuilder

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
```

Comment:
I don't believe this file has anything to do with the specific change.

Reply:
Done!